test/index.js

```diff
@@ -835,8 +835,67 @@ describe('Signal K SDK', () => {
 
       client.connect().catch((err) => done(err))
     }).timeout(20000)
+
+    it('... Send ping in WS every 1s and wait for 10 delta update "navigation.datetime"', (done) => {
+      const client = new Client({
+        hostname: 'demo.signalk.org',
```

Comment:
I am not terribly fond of using demo server for testing, I much prefer self contained tests, but we lost that particular ball long time ago, so this is probably better than nothing.